openff/cli/tests/test_cli_functions.py

```diff
@@ -0,0 +1,330 @@
+import pathlib
+import tempfile
+
+import numpy as np
+import pytest
+from openforcefield.topology import Molecule
+from openforcefield.typing.engines.smirnoff.forcefield import get_available_force_fields
+from openforcefield.utils import (
+    OPENEYE_AVAILABLE,
+    RDKIT_AVAILABLE,
+    OpenEyeToolkitWrapper,
+    RDKitToolkitWrapper,
+    temporary_cd,
+)
+
+from openff.cli.check_versions import get_versions
+from openff.cli.generate_conformers import (
+    generate_conformers,
+    make_registry,
+    write_mols,
+)
+from openff.cli.tests.utils import get_data_file_path
+
+# TODO: Run all tests in a safe temporary directory
+
+
+class TestCheckVersions:
+    def test_basic_output(self):
+        out = get_versions()
+        from openforcefield import __version__ as toolkit_version
+
+        from openff.cli import __version__ as cli_version
+
+        assert "OpenFF Toolkit" in out
+        assert "OpenFF CLI" in out
+        assert toolkit_version in out
+        assert cli_version in out
+
+
+# TODO: Update skipifs after #615
+@pytest.mark.skipif(not RDKIT_AVAILABLE, reason="requires rdkit")
+@pytest.mark.skipif(not OPENEYE_AVAILABLE, reason="requires openeye")
+@pytest.mark.parametrize("toolkit", ["rdkit", "openeye"])
```

Comment:
(blocking) Could we add a simple test that runs on an RDKit-only build? I would just want to make sure that the workflow isn't quietly reliant on some little OE-dependent funcitonality.

Reply:
I've updated the pytest setup so this class's tests actually run  on the RDKit-only builds (after #10). Will check the build logs later but it's working as expected locally

```
======================================== 41 passed, 42 skipped, 1 warning in 136.18s (0:02:16) =========================================